src/events.ts

```diff
@@ -0,0 +1,36 @@
+export const EVENTS = {
+  // Record when a user succesfully logs in.
+  LOGIN_SUCCEEDED: '$login.succeeded',
```

Comment:
I still feel this should be made as structs for further future development

Reply:
It's on the list for improvements, but I think it's time to downscope this PR :)